src/containers/EventListeners/EventListeners.js

```diff
@@ -0,0 +1,220 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import isEqual from 'lodash.isequal';
+import { injectIntl } from 'react-intl';
+import { getErrorMessage } from '@tektoncd/dashboard-utils';
+import { InlineNotification } from 'carbon-components-react';
+import {
+  FormattedDate,
+  LabelFilter,
+  Table
+} from '@tektoncd/dashboard-components';
+import { fetchEventListeners } from '../../actions/eventListeners';
+import {
+  getEventListeners,
+  getEventListenersErrorMessage,
+  getSelectedNamespace,
+  isFetchingEventListeners,
+  isWebSocketConnected
+} from '../../reducers';
+
+import '../TriggerTemplates/TriggerTemplates.scss';
```

Comment:
Hmm what's the deal with this? Shouldn't we have a shared scss across all related components or at least EventListeners.scss?